### ISSUE TITLE
Loading TIPI.TIPICFG hangs the TI for a few folks.

### ISSUE THREAD

**Issue by jedimatt42** [jedimatt42/tipi#77]
Greg said he wasn't patient enough... 
Leonard said it hung, and then after a reboot of the 4a it was fine.

**Comment by jedimatt42** [comment]
We have seen a hang as recent as Mar 29th 2018.

Hangs today seem to be due to noise on the lines connecting the PI to the TIPI board.  I don't have a good solution for this. 

We have added a parity bit to the communication protocol that should retry if a single bit ( or odd number of bits ) error occurs.  This fixed all errors for me in my 'clean' environment. 

Next, Greg still had lots of errors.. But eliminating / trading speech synthesizers, removed most. 

Third, we lowered the transfer rate from 8k/sec to 6k/sec.  48000bps equivalent.  This has solved the issue in Greg's 'noisy' environment.  

How 'clean' and how 'noisy' are not really known... It was not measured.   PI-Zero-W with a more direct link to the back of the TIPI board show results consistent with this theory, having a much shorter signal line, and being slower, we don't see noise related errors.
